### ISSUE TITLE
NetCDF3 NC GIOPS Forecast

### ISSUE THREAD

**Issue by NoahGallant-MUN** [DFO-Ocean-Navigator/Ocean-Data-Map-Project#401]
You currently can't create a NetCDF3-NC subset for giops_forecast.
From what I can tell this is because the lats and longitudes have different shapes? (that is the error I'm getting)
Norm has requested this.

**Comment by NoahGallant-MUN** [comment]
This seems to be because GIOPS forecast has lon lat, and GIOPS daily has x and y coordinates?

**Comment by htmlboss** [comment]
Yeah if I remember correctly, forecast is on the user grid (lon, lat) and most of the others are on the native grid (x, y).
